mock/src/components/Commands.ts

```diff
@@ -0,0 +1,97 @@
+import { REPLFunction } from "./REPL";
+import {
+  datamocked,
+} from "./mockedJson";
+
+/**
+ * Map to store registered commands.
+ */
+export const commandRegistry = new Map();
```

Comment:
We suggest having a separate class just for the map, and creating add and delete functions for developer use as a more defensive strategy, whereas commandRegistry is currently denoted as "export".